### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by DarkMatter12000** [jason0611/FS22_DashboardLive#86]
Is there a way to get the DBL Map to display fertilizer, cultivation, Plow states as well as update the map when I connect two fields by plowing?
Also is there a way I can get it to show my CoursePlay Routes?

**Comment by jason0611** [comment]
No, there is no way to put this informations into the "3D-World" of the game in FS22. There is no shader for it and the game's engine doesn't support it. 
Maybe it's possible in FS25, but before I can check this, I have to finish the migration of DBL to the new version of the game.
